### DISCUSSION TITLE
1st March to 31st March Report - Furqan Ahmed

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#123] [General]
# Work Summary
This is in continuation with the [previous report](https://github.com/orgs/dcdao/discussions/101), in request of the steering committee for the month of March 2024.

## Request Integration to OKX Wallet
- **Objective**: Support Darwinia/Crab chain by default.
- **Action**: Needs to be taken through listing process using an official domains email. Ranji has been requested to handle this.

## Fixing RPC for Dsafe
- **Challenge**: The repository was over 2 years old, and WalletConnect stopped supporting `brdge.walletconnect` URL, causing our product to malfunction.
- **Solution**: More feasible to integrate a new fork than to fix the old one. Spent a day debugging, deciding, and integrating.
- **Outcome**: Yalin (fewsena) updated the support repos. Now operational.
- **Link**: [Dsafe Multisig Wallet](https://multisig.darwiniacommunitydao.xyz/welcome)

## Completing and Reviewing "Get Ring" Page for Darwinia Website
- **Completion**: Fulfilled design and requirements.
- **Links**: 
  - [Merged Pull Request](https://github.com/darwinia-network/home/pull/381)
  - [Related Issue](https://github.com/darwinia-network/home/issues/370)
  - Multiple Smaller Issues Fixed: 
  [Usability Issues](https://github.com/darwinia-network/home/pull/397)
  [Minor Issue 1](https://github.com/darwinia-network/home/pull/394)
  [Minor Issue 2](https://github.com/darwinia-network/home/pull/389)
  
## Request Integration to 1PRC
- **Action**: Requested integration, currently awaiting response.

## Integrate Helixbridge.app with Li.Fi
- **Interaction**: Discussed with Sarthak from Li.Fi.
- **Feedback**: No new bridge integrations until Q2. Will be contacted by Li.Fi team for integration.

## Integration with Rabby Wallet, Support Darwinia Chain
- **Process**: Discussed with Eleanor from Debank. Created a proposal for listing.
- **Requirement**: Minting of web3ID for 96 USD, which was requested from the team.
- **Progress**: Received funds, submitted proposal, received votes and backing of 1.5 million, but ultimately rejected.
- **Links**:
  - [Proposal](https://debank.com/proposal/65244)
  - [Discussion](https://github.com/orgs/dcdao/discussions/103#discussioncomment-8879942)

## Darwinia Token 2049 Dubai Side Event Proposal
- Started a Temperature check discussion for Darwinia's Side event in Token 2049 Dubai. Read More [here](https://github.com/orgs/dcdao/discussions/122)

## Other Activities
- **Doing base works for the foundation of Technical Delegation & Management**
- **Human Resources**: Interviewed a candidate for an analyst role. Looking for more eligible candidates, working with @wuminzhe on specifics about the role.
- **Development, Management, and Maintenance DCDAO Proposal**: Created and shared for feedback. [View Proposal](https://github.com/orgs/dcdao/discussions/121)
- **Rabby** introduced a new feature to custom add "Testnets"(can also add mainnets) - So now you can simply add Dawinia and Crab directly from Chainlist; More [here](https://github.com/orgs/dcdao/discussions/103#discussioncomment-8963904)

CC: @hackfisher @wuminzhe

**Comment by hackfisher** [comment]
Thanks for updating.
